src/com/lftechnology/java/training/srijan/basics/coreapp/controller/EmployeeManagementController.java

```diff
@@ -0,0 +1,186 @@
+package com.lftechnology.java.training.srijan.basics.coreapp.controller;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.Employee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+import com.lftechnology.java.training.srijan.basics.coreapp.service.EmployeeServiceImpl;
+import com.lftechnology.java.training.srijan.basics.coreapp.service.ServiceFactory;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.List;
+import java.util.Scanner;
+
+public class EmployeeManagementController {
+	private static final Logger LOGGER = Logger.getLogger(EmployeeManagementController.class.getName());
+	List<Employee> employees;
+	Employee employee;
+	Employee e = new Employee();
+	EmployeeServiceImpl employeeServiceImpl = ServiceFactory.getEmployeeService();
+
+	public EmployeeManagementController() {
+	}
+
+	public UserRole save(String userName, String password) {
+		e.setUserName(userName);
+		e.setPassword(password);
+		return employeeServiceImpl.save(e);
+	}
+
+	public void adminUserOptionHandler(int adminChoice) {
+		switch (adminChoice) {
+		case 1:
+			Employee employeeDetails = inputUserInfo(e);
+			employeeServiceImpl.addEmployee(employeeDetails);
+			break;
+		case 2:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this switch case number of lines from 9 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1151)